docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+db:
+  image: postgres
+  environment:
+    POSTGRES_USER: postgres
+    POSTGRES_PASSWORD: postgres
+web:
+  image: kinto/kinto-server
```

Comment:
This should be the: webpush-channels image probably but you need to publish it first.